HHS/Head-Start-TTADP
# [Prod] Fix bug with resources not saving, add goal status explanation modal, add migration to remove duplicate legacy topics

## Description of change

- Fix bug where resources were sometimes not properly saved
- Add modal to goals & objectives tab explaining goal statuses
- Add a migration to remove duplicate legacy topics
- Also updated dependencies on the backend

## How to test

- Save resources, ensure they are all saved properly
- Also test the goal status explanation modal

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-850
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-918


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
